libseccomp/src/filter_context.rs

```diff
@@ -594,6 +730,15 @@ impl ScmpFilterContext {
     ///
     /// If this function is called with an invalid filter or an issue is
     /// encountered setting the attribute, an error will be returned.
+    ///
+    /// # Examples
+    ///
+    /// ```
+    /// # use libseccomp::*;
+    /// let mut ctx = ScmpFilterContext::new_filter(ScmpAction::Allow)?;
+    /// ctx.set_ctl_nnp(true)?;
```

Comment:
```suggestion
    /// ctx.set_ctl_nnp(false)?;
```

Reply:
Added the fix in [commit](https://github.com/libseccomp-rs/libseccomp-rs/commit/2e07e17f07f92e2b6f7766c3c2d67495dc762efe)